src/cargo_toml.rs

```diff
@@ -144,34 +104,22 @@ struct CargoTomlMetadata {
 /// Those fields are the part of the
 /// [python core metadata](https://packaging.python.org/specifications/core-metadata/)
 /// that doesn't have an equivalent in cargo's `[package]` table
-#[derive(Serialize, Deserialize, Debug, Clone, Eq, PartialEq, Default)]
+#[derive(Serialize, Deserialize, Debug, Clone, Default)]
 #[serde(rename_all = "kebab-case")]
-#[serde(deny_unknown_fields)]
```

Comment:
Had to remove this to deserialize unknown fields to `other` and print a warning for it.